samples/word/05-scenarios/content-generation.yaml

```diff
@@ -0,0 +1,507 @@
+    language: typescript
+template:
```

Comment:
The task pane looks very different from other Script Lab samples. Please use the same layout and style as other samples.